spec/controllers/state_file/questions/az_qualifying_organization_contributions_controller_spec.rb

```diff
@@ -9,6 +9,51 @@
     sign_in intake
   end
 
+  describe ".show?" do
+
+    context "when the intake has charitable contributions" do
+      before do
+        intake.charitable_contributions_yes!
+      end
+
+      context "when the intake has cash contributions" do
+
+        before do
+          intake.update(charitable_cash_amount: 1)
+        end
+
+        it "shows" do
+          expect(described_class).to be_show(intake)
```

Comment:
i feel like this is the first time I've seen the use of `be_show` 😮

Reply:
it's kind of obnoxious - i wrote `expect(described_class.show?(intake).to be_truthy` and this was rubocop's suggested replacement, just because `show?` ends with a question mark. but, i didn't know you could pass arguments to the magic `be_x` matchers that would get passed through to the underlying methods, so we all learned something today